src/applications/edu-benefits/tests/0994/e2e/vet-tec-helpers.js

```diff
@@ -23,6 +23,7 @@ export const completeAlreadySubmitted = (client, data) => {
     `root_appliedForVaEducationBenefits`,
     _.get(data, 'appliedForVaEducationBenefits', false) ? 'Y' : 'N',
   );
+  client.pause(1000);
```

Comment:
It's tough to tell what this is waiting for when it's put at the end of a helper. I think we generally prefer something like [`waitForElementVisible`](http://nightwatchjs.org/api#waitForElementVisible) that documents what this is waiting for or to use a wait command _before_ the action. 

The e2e tests are executed serially so this adds another second to the overall build time on Jenkins.